R/load_fb.R

```diff
@@ -30,7 +30,9 @@
 #' }
 
 load_match_results <- function(country, gender, season_end_year, tier) {
-  dat_urls <- paste0("https://github.com/JaseZiv/worldfootballR_data/blob/master/data/match_results/", country, "_match_results.rds?raw=true")
+  "https://github.com/JaseZiv/worldfootballR_data/releases/download/match_results/ARG_match_results.rds"
```

Comment:
looks like test code 😄

Reply:
It absolutely was! Embarrassing! Thanks for picking up. Rectified now